.copr/Makefile

```
@@ -1,5 +1,6 @@
 BUILDROOT:=$(shell [ -d "/build" ] && echo "/build" || echo ".")
 TMPDIR:=$(shell mktemp -d)
+TARGET:=target
```

```suggestion
TARGET:=$(or $(CARGO_TARGET_DIR),target)
```

And don't duplicate TARGET in https://github.com/whamcloud/integrated-manager-for-lustre/pull/1881/files#diff-76ac15dc8b7c623d9556dee53bdd8b34R124

---

Done